src/main.lua

```diff
@@ -2610,4 +2610,21 @@ addEventingHandler("allPaginatedGatewayVaults", utils.hasMatchingTag("Action", "
 	Send(msg, { Target = msg.From, Action = "All-Gateway-Vaults-Notice", Data = json.encode(result) })
 end)
 
+-- Temporary handler for posting epoch distribution notices (make sure to remove this once previous epochs are distributed)
+addEventingHandler("postEpochDistributionNotice", utils.hasMatchingTag("Action", "Post-Epochs"), function(msg)
+	-- iterate over epochs and post epoch-distribution-notice for each epoch
+	for epochIndex in pairs(epochs.getEpochs()) do
```

Comment:
Does the linter remove `, _` if you were to specify it for the unused values?

Reply:
it's indifferent, added here for verbosity [c229f56](https://github.com/ar-io/ar-io-network-process/pull/328/commits/c229f56199a82931cd0951f2efbc643a6add8525)